renovate.json

```diff
@@ -20,6 +21,9 @@
   "labels": [
     "dependencies"
   ],
+  "supportPolicy": [
+    "lts_active"
```

Comment:
See https://renovatebot.com/docs/node/#configuring-support-policy for more information on this one.

The tl;dr is this specifies we are only interested in upgrades to LTS versions.

Even more information at https://github.com/nodejs/Release#release-schedule.

Reply:
I've updated this to the `lts` policy, as we're happy under the maintenance period too.